pkgs/os-specific/linux/brillo/default.nix

```diff
@@ -0,0 +1,28 @@
+{stdenv, fetchFromGitLab , which, go-md2man, coreutils}:
+
+stdenv.mkDerivation rec {
+  version = "1.4.3";
+  name = "brillo-${version}";
+  src = fetchFromGitLab {
+    owner= "cameronnemo";
+    repo= "brillo";
+    rev= "v${version}";
+    sha256 = "1syv3iav7bwr84x9frz1qd6qmgp8ldbjs4gl3r94nhllkai9spaq";
+  };
+  makeFlags = [ "PREFIX=" "DESTDIR=$(out)"];
```

Comment:
Please do not use `DESTDIR`, it makes the path opaque to the project. `"PREFIX=${placeholder ''out''}"` should be enough. If something does not work such as AppArmor directory, you can override it individually.

Reply:
are you suggesting I patch the makefile to change the AppArmor directory?

Reply:
Changing it in `makeFlags` should work here.

Reply:
The AppArmor directory is set to `$(DESTDIR)/etc/apparmor.d` though so surely I can't change this without either patching or setting `DESTDIR` (I could just be misunderstanding makefiles though)

Reply:
Variables set in `Makefile` will be still overwritten by those passed to `make`.